packages/nodes-base/nodes/Hubspot/DealDescription.ts

```diff
@@ -778,7 +778,7 @@ export const dealFields = [
 					loadOptionsMethod: 'getDealProperties',
 				},
 				default: [],
-				description: `Used to include specific deal properties in the results. By default, the results will only include Deal ID and will not include the values for any properties for your companys. Including this parameter will include the data for the specified property in the results. You can include this parameter multiple times to request multiple properties separated by ,.`,
+				description: `<p>Used to include specific deal properties in the results. By default, the results will only include company ID and will not include the values for any properties for your company.</p><p>Including this parameter will include the data for the specified property in the results. You can include this parameter multiple times to request multiple properties separated by a comma: <code>,</code>.</p>`,
```

Comment:
was the change from `Deal` to `company` intentional? not familiar enough with this node to know

Reply:
Copy-paste artifact